egs/chime4/s5_1ch_track/local/chime4_train_rnnlms.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
OK, will do it tonight.

On Mon, Apr 25, 2016 at 1:09 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/chime4/s5_1ch_track/local/chime4_train_rnnlms.sh
> <https://github.com/kaldi-asr/kaldi/pull/730#discussion_r60950084>:
>
> > @@ -0,0 +1,111 @@
> > +#!/bin/bash
>
> @hainan-xv <https://github.com/hainan-xv> could you please review this
> script.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/730/files/975abc99896ff7b0c5f6dced6109fc31cb40d34e#r60950084>
>



-- 
- Hainan